src/GravityForms/GravityFormsAddon.php

```diff
@@ -80,113 +78,203 @@ public function plugin_settings_fields()
     {
         return [
             $this->settingsGeneral(),
+            $this->settingsDescription(),
             $this->settingsOpenZaak(),
             $this->settingsDecosJoin(),
-            $this->settingsEnableU(),
-            $this->RSIN()
+            $this->settingsRxMission(),
+            $this->settingsXxllnc(),
         ];
     }
 
     protected function settingsGeneral(): array
     {
         return [
-            'title'  => esc_html__('Description', config('core.text_domain')),
-            'description' => esc_html__(' Enter the details of the suppliers you would like to use.', config('core.text_domain')),
+            'title' => esc_html__('Algemene instellingen', 'owc-gravityforms-zaaksysteem'),
+            'fields' => [
+                [
+                    'label' => esc_html__('RSIN-nummer', 'owc-gravityforms-zaaksysteem'),
+                    'type' => 'text',
+                    'class' => 'medium',
+                    'name' => "{$this->prefix}-rsin",
+                    'description' => 'Registratienummer voor organisaties. Wordt door (overheids)organisaties gebruikt om informatie met elkaar uit te wisselen.',
+                ],
+                [
+                    'label' => esc_html__('Zaak afbeelding', 'owc-gravityforms-zaaksysteem'),
+                    'type' => 'text',
+                    'name' => "{$this->prefix}-zaak-image",
+                    'description' => esc_html__('Stel de header-afbeelding in voor een zaak. Laat leeg voor een standaard grijze achtergrond.'),
+                ],
+            ],
+        ];
+    }
+
+    protected function settingsDescription(): array
+    {
+        return [
+            'title' => esc_html__('Toelichting', 'owc-gravityforms-zaaksysteem'),
+            'description' => esc_html__('Voer de gegevens in van de leveranciers die je wilt gebruiken. Voeg geen endpoint-type toe, dus "https://supplier.com/api/v1" in plaats van "https://supplier.com/api/v1/eigenschappen".', 'owc-gravityforms-zaaksysteem'),
             'fields' => [[]],
```

Comment:
Deze hele `settingsDescription()` mag verwijderd worden. De empty `fields` array zorgt in `plugins/gravityforms/includes/addon/class-gf-addon.php` voor problemen: er wordt namelijk overheen geloopt, en vervolgens missen er keys. Zie screenshot hieronder. 

![Screenshot 2023-12-12 at 13 55 55](https://github.com/OpenWebconcept/plugin-owc-gravityforms-zaaksysteem/assets/48315669/340b93d3-99dd-4c9c-9189-5c9cde8037ae)